From f926adc3309e7e82999e6db8fbb959eca82afea2 Mon Sep 17 00:00:00 2001
From: Nate Graham <nate@kde.org>
Date: Wed, 2 Oct 2019 14:05:57 -0600
Subject: [PATCH 07/12] [KCM] Make the warning message wider

Summary:
Making the message wider improves the presentation, especially for wordy languages
where the message may become very tall.

Test Plan: {F7500913}

Reviewers: #vdg, #plasma, romangg, dhaumann, ndavis

Reviewed By: #vdg, dhaumann, ndavis

Subscribers: dhaumann, plasma-devel

Tags: #plasma

Differential Revision: https://phabricator.kde.org/D24371
---
 kcm/package/contents/ui/Panel.qml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kcm/package/contents/ui/Panel.qml b/kcm/package/contents/ui/Panel.qml
index 2ecaad8..294a21e 100644
--- a/kcm/package/contents/ui/Panel.qml
+++ b/kcm/package/contents/ui/Panel.qml
@@ -109,6 +109,7 @@ ColumnLayout {
 
         Kirigami.InlineMessage {
             id: weirdScaleFactorMsg
+            Kirigami.FormData.isSection: true
             Layout.fillWidth: true
             type: Kirigami.MessageType.Warning
             text: i18n("Scale factors that are not a multiple of 0.25 may cause visual glitches in applications. Consider setting the scale factor to a multiple of 0.25 and adjusting the font size instead.")
-- 
2.21.0

